OpTestConfiguration.py

```diff
@@ -304,6 +304,12 @@ def get_parser():
         "--host-cmd-timeout", help="Timeout for command", type=int, default=1000)
     hostcmdgroup.add_argument("--host-cmd-resultpath",
                               help="Result path from host", default="")
+    hostcmdgroup.add_argument("--lpar-config",
+                              help="lpar configuration values, "
+                                   "Ex: --lpar-config=\"cpu=dedicated,vtpm=1,vpmem=1\"")
+    hostcmdgroup.add_argument("--cec-config",
+                              help="CEC configuration values, "
+                              "Ex:--cec-config=\"lmb=2GB,hugepage=16GB\"")
```

Comment:
why 2 parameters   why not a single param --machine-config

which can take all arguments and the testcase should decide which one is cec and lpar

Reply:
i think we need to go with 2 params that helps to handle things easily and later we can add os as another param

Reply:
@PraveenPenguin my point is, we can still keep separate classes for lpar, cec and os config... but only single --macine-config parameter please.. so it is easy for framework ..as it already the parameters are grown like anything

Reply:
there will be tests suite

```
if lmb in conf.get.machine_config:
addtest  MachineConfig().CEC_config:
if hugepage in conf.get.machine_config:
addtest  MachineConfig().os_config:
if vtpm in conf.get.machine_config:
addtest MachineConfig().hmc_config:
```

Reply:
or inisde the test itself you can parse based on parameters given